gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/instrumentation/reaper/ReaperClassLoadTransform.kt

```diff
@@ -10,15 +10,28 @@ import org.objectweb.asm.Opcodes
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import java.security.MessageDigest
+import java.util.Base64
+import java.io.PrintWriter
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.Internal
+
+interface ReaperTransformParameters : InstrumentationParameters {
+  @get:Internal
```

Comment:
Can we make this an output file so this is tracked properly with Gradle? I’m not actually sure how the params are wired in to AGP transforms but I believe they should do that.
```suggestion
  @get:OutputFile
```